include/ignition/rendering/base/BaseCamera.hh

```diff
@@ -286,6 +286,7 @@ namespace ignition
     void BaseCamera<T>::SetImageWidth(const unsigned int _width)
     {
       this->RenderTarget()->SetWidth(_width);
+      this->SetAspectRatio(1.0 * _width / this->ImageHeight());
```

Comment:
I assume you are multiplying by `1.0` to cast the result to a `double`. It's better to use `static_cast<double>` to be explicit with the cast making the purpose clear. Same for `SetImageHeight`
```suggestion
      this->SetAspectRatio(
        static_cast<double>(_width) / static_cast<double>(this->ImageHeight()));
```